sqlglot/expressions.py

```diff
@@ -5439,6 +5416,34 @@ def maybe_copy(instance, copy=True):
     return instance.copy() if copy and instance else instance
 
 
+def _to_s(node: t.Any, hide_missing: bool = True, level: int = 0) -> str:
+    """Generate a textual representation of an Expression tree"""
+    if isinstance(node, Expression):
+        args = {k: v for k, v in node.args.items() if v is not None or not hide_missing}
+        if node.comments or not hide_missing:
+            args["comments"] = node.comments
+        if node.type or not hide_missing:
+            args["type"] = node.type
+
+        # Inline Expressions that don't have any child Expressions.
+        # This helps make expressions like Identifier more compact.
+        if any(isinstance(v, (Expression, list, tuple)) for v in args.values()):
```

Comment:
i don't think a child can be a tuple, only a list. this method actually seems useful on base expression, it could be called is_leaf or something... wonder if this could be used elsewhere

Reply:
Ah, I was just copying the logic of `ensure_list`. But I can change it to just `list`.